test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+ansible==2.4.5.0
```

Comment:
Shouldn't be 2.3.1.0?

Reply:
You're right, but the problem is that Molecule >= 2.18 needs Ansible >= 2.4 and MySQL role as well. I'm not sure which is the best solution, but I'm thinking of upgrading the Ansible minimum version, create an issue for updating all the role to Ansible 2.4 (`import_tasks` and so on)... and release a major version :/

Reply:
IMHO In that case the Readme file (and _meta/main.yml_) should specify Ansible 2.4.5.0 (the version used in our tests)

Reply:
Done